Source/WTF/wtf/WTFConfig.cpp

```diff
@@ -73,9 +73,13 @@ void setPermissionsOfConfigPage()
     std::call_once(onceFlag, [] {
         mach_vm_address_t addr = std::bit_cast<uintptr_t>(static_cast<void*>(WebConfig::g_config));
         auto flags = VM_FLAGS_FIXED | VM_FLAGS_OVERWRITE | VM_FLAGS_PERMANENT;
+        auto prot = VM_PROT_READ | VM_PROT_WRITE;
 
         auto attemptVMMapping = [&] {
-            return mach_vm_map(mach_task_self(), &addr, ConfigSizeToProtect, pageSize() - 1, flags, MEMORY_OBJECT_NULL, 0, false, VM_PROT_READ | VM_PROT_WRITE, VM_PROT_READ | VM_PROT_WRITE, VM_INHERIT_DEFAULT);
+            // We use remap here to avoid zeroing the config, since WebKit
```

Comment:
nit: should specify "remap _instead of map_", since I think many people aren't aware that mach_vm_map/mmap will zero memory.